courtlistener/process_cl.py

```diff
@@ -0,0 +1,132 @@
+import argparse
+import csv
+import logging
+import os
+import re
+import sys
+
+import pandas as pd
+
+from licensed_pile.licenses import PermissiveLicenses
+from licensed_pile.logs import configure_logging
+from licensed_pile.write import to_dolma
+
+SOURCE_NAME = "CourtListenerOpinion"
+
+csv.field_size_limit(sys.maxsize)
```

Comment:
Is this needed now that we aren't using the csv stdlib?

If it isn't there are several unused imports to remove.

Reply:
Deleted this and removed imports.